### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kinow**
Hi! Few minor typos. I think the only more important is the main comment in `payu/__init__.py` which may introduce Payu to users 👍

### CODE REVIEW

File: docs/source/config.rst [hunk 1]
```diff
@@ -147,7 +147,7 @@ configuration.
 
 ``exe``
    Binary executable for the model. This can either be a filename in the
-   laboratorys ``bin`` directory, or an absolute filepath. Various model
+   laboratories ``bin`` directory, or an absolute filepath. Various model
```
Comment (marshallward) [inline docs/source/config.rst]: laboratory's

File: payu/manifest.py [hunk 2]
```diff
@@ -162,7 +162,7 @@ def add_filepath(self, filepath, fullpath, copy=False):
     def add_fast(self, filepath, hashfn=fast_hashes, force=False):
         """
         Bespoke function to add filepaths but set shortcircuit to True, which
-        means only the first calculatable hash will be stored. In this way only
+        means only the first calculable hash will be stored. In this way only
```
Comment (marshallward) [inline payu/manifest.py]: (seems both are valid but smaller is better :D )

**Comment by marshallward** [comment]
Thanks, especially for hunting the repeated typos down :)

File: payu/manifest.py (hunk 2 above)
Comment (kinow) [reply payu/manifest.py]: Oh! Today-I-Learned! Thanks @marshallward !

**Comment by kinow** [comment]
Thank you too @marshallward !

**Pull request merged**